idl/storage_service.idl.hh

```diff
@@ -36,6 +36,6 @@ struct raft_topology_snapshot {
 
 struct raft_topology_pull_params {};
 
-verb raft_topology_cmd (raft::term_t term, service::raft_topology_cmd) -> service::raft_topology_cmd_result;
```

Comment:
The commit log is missing explanation why do we need to prevent RPC reordering.

Reply:
fixed